<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sensor_edit</class>
 <widget class="QWidget" name="Sensor_edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sensor_edit</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="TODO" native="true">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="MatplotlibWidget" name="sensor_diagram" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>460</width>
       <height>460</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MatplotlibWidget</class>
   <extends>QWidget</extends>
   <header>MatplotlibWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
